### PR TITLE
#6691 default value for radio button elements

### PR DESCRIPTION
**Pull request by jersiovic**
Furthermore I have fixed a bug that makes that after a postback a checked radio button is rendered as unchecked

### CODE REVIEW

**Comment by sebastienros** [comment]
While we review the change, please squash the changes.

File: src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/RadioButtonElementDriver.cs [hunk 1]
```diff
@@ -34,8 +34,14 @@ public RadioButtonElementDriver(IFormsBasedElementServices formsServices, IToken
                         Name: "Value",
                         Title: "Value",
                         Classes: new[] { "text", "medium", "tokenized" },
-                        Description: T("The value of this radio button.")));
-
+                        Description: T("The value of this radio button.")),
+                    _DefaultValue:
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/RadioButtonElementDriver.cs]: Formatting

**Comment by sebastienros** [comment]
Sign-off checklist:
[ ] - Grammar (wording and translations)
[ ] - It works

**Comment by jersiovic** [comment]
I'm new with Git. When you say I have to squash I understand you want I fuse all in one commit. Those previous commits that appear are: first two a test from a collegue in our 1.10 branch. The others are merges from Orchard repo 1.10 and 1.10.x branch to our repo 
I have tried to make those to disappear with rebase with autosquash option without success. After different attempts with rebase and reset to this commit I see I have increased the number of files changed for this pull request, with changes that shouldn't be there. So, I'm thinking on close this pull request, resetting our repo and resubmit you another pull request with a more simple scenario. Next for sure I will need to start to read a good book about git to avoid problems of doing things like that.

**Comment by jersiovic** [comment]
I've seen in another github project that Branching Strategy and Forking guidelines https://github.com/01org/media-service-upnp/wiki/Branching-Strategy-and-Forking talk about a rebase step I didn't do after merge changes to branches in our repo, that maybe is the key why those extra commits were visble in the pull request:
I copy paste what it is interesting on that page, can you confirm me it should be done in the same wat for Orchard contributions? I haven't find a concrete explanation like that in orchard guidelines http://docs.orchardproject.net/Documentation/Contributing-patches

...
Then fetch the latest changes and merge them into your master branch

git fetch upstream
git checkout master
git pull upstream master
Next you need to rebase your development branch with the latest changes you just received

git checkout dev-011
git rebase master
Then push your new branch to your fork's origin and submit a pull request

git push origin dev-011
It could happen that multiple pull request have been sent by different developers at the same time. If this occurs, and another pull request is merged first, your pull request might be rejected as it will no longer be possible to perform a fast forward merge from your development branch. If this happens, sync your local repository once more, create a new branch based on the latest sources and move your changes to this branch, either by using rebase onto or cherry pick.
..

File: src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/RadioButtonElementDriver.cs (hunk 1 above)
Comment (jersiovic) [inline src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/RadioButtonElementDriver.cs]: I don't see what's wrong with formatting. _DefaultValue var is at the same level of indentation of its sibling var Description. 
Do you mean '_' char? I saw other elements using a '_' char before the name of the variable. Do I have to rename Description to _Description or _DefaultValue to DefaultValue ?

**Comment by jersiovic** [comment]
I have open this other pull request with only one commit https://github.com/OrchardCMS/Orchard/pull/6697
So, I close this one.

**Pull request closed without merging**